ql/src/java/org/apache/hadoop/hive/ql/ddl/table/drop/DropTableOperation.java

```diff
@@ -109,7 +109,7 @@ public int execute() throws HiveException {
     }
 
     // TODO: API w/catalog name
-    context.getDb().dropTable(desc.getTableName(), desc.isPurge());
+    context.getDb().dropTable(table, desc.isPurge());
```

Comment:
nit: Why is this change needed?

Reply:
I need to pass extra info like txnId through the tbl object